CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.12.5
+------
+
+- fix: `AuthorizeGroupEgress` could return `authorizeGroupIngress` as name
```

Comment:
I don't unterstand why the changelog mentions `authorizeGroupIngress`.

Reply:
`Authorize..` is the go struct, `authorize...` is the CloudStack API call.